api/src/main/java/org/apache/brooklyn/api/objs/Configurable.java

```diff
@@ -41,6 +41,11 @@
     @Deprecated
     public <T> T setConfig(ConfigKey<T> key, T val);
 
+    /**
+     * Convenience method for {@code config().get(key)}
+     */
+    <T> T getConfig(ConfigKey<T> key);
```

Comment:
`config().get()` is not much longer, so this is of questionable value. But if we have it, better have the mirroring method as well - remove `@Deprecated` from the `setConfig`.

Personally I prefer to have one true way to do things (in general, not only here).